lib/admin.php

```diff
@@ -338,5 +338,73 @@ function woocommerce_embed_page_header() {
 	</div>
 	<?php
 }
-
 add_action( 'in_admin_header', 'woocommerce_embed_page_header' );
+
+/**
+ * Registers WooCommerce specific user data to the Wordpress user API.
+ */
+function wc_admin_register_user_data() {
+	register_rest_field(
+		'user',
+		'woocommerce_meta',
+		array(
+			'get_callback'    => 'wc_admin_get_user_data_values',
```

Comment:
These array items should be aligned per the coding standards. https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#indentation

Reply:
Fixed in 8ba7438